src/plugins/discover/public/application/apps/main/services/use_discover_state.ts

```diff
@@ -29,24 +28,23 @@ import { SortPairArr } from '../components/doc_table/lib/get_sort';
 export function useDiscoverState({
   services,
   history,
-  initialIndexPattern,
   initialSavedSearch,
```

Comment:
🧹 `initialSavedSearch` is just assigned to `savedSearch` in L43 ... so it should just be named `savedSearch`, also it doesn't need to be exported from this file